incubator/hnc/e2e/tests/namespace_test.go

```diff
@@ -10,57 +10,47 @@ import (
 var _ = Describe("Namespace", func() {
 	var (
 		prefix string
+		nsA    string
+		nsB    string
 	)
```

Comment:
I'd just make these constants and get rid of `BeforeEach`:

```
const (
  prefix = namespacePrefix+"namespace-"
  nsA = prefix+"a"
  nsB = prefix+"b"
)
```

Reply:
done